acquisitions/src/main/resources/thunderjet/cross-modules/features/rollover-with-pending-order.feature

```diff
@@ -0,0 +1,161 @@
+      * def budgetId1 = call uuid
+      * def budgetId2 = call uuid
+      * def orderId = call uuid
+      * def poLineId = call uuid
+      * def rolloverId = call uuid
+
+      ## Create fiscal years and associated ledgers
+      * def fromYear = call getCurrentYear
+      * def toYear = parseInt(fromYear) + 1
+      * def periodStart1 = fromYear + '-01-01T00:00:00Z'
+      * def periodEnd1 = fromYear + '-12-30T23:59:59Z'
+      * def series = 'TESTFYC'
+      * def v = call createFiscalYear { id: '#(fyId1)', code: '#(series + "0001")', periodStart: '#(periodStart1)', periodEnd: '#(periodEnd1)', series: '#(series)' }
+      * def periodStart2 = toYear + '-01-01T00:00:00Z'
+      * def periodEnd2 = toYear + '-12-30T23:59:59Z'
+      * def v = call createFiscalYear { id: '#(fyId2)', code: '#(series + "0002")', periodStart: '#(periodStart2)', periodEnd: '#(periodEnd2)', series: '#(series)' }
+      * def v = call createLedger { id: '#(ledgerId)', fiscalYearId: '#(fyId1)' }
+
+
+      ## Create fund and budgets
+      * configure headers = headersAdmin
+      * def v = call createFund { id: '#(fundId)', code: '#(fundId)', ledgerId: '#(ledgerId)' }
+      * def v = call createBudget { id: '#(budgetId1)', fundId: '#(fundId)', fiscalYearId: '#(fyId1)', allocated: 100, status: 'Active' }
+      * def v = call createBudget { id: '#(budgetId2)', fundId: '#(fundId)', fiscalYearId: '#(fyId2)', allocated: 100, status: 'Active' }
+      * configure headers = headersUser
+
+
+      ## Create the order and line
+      * def ongoing = { interval: 123, isSubscription: true, renewalDate: '2022-05-08T00:00:00.000+00:00' }
+      * def v = call createOrder { id: '#(orderId)', orderType: 'Ongoing', ongoing: '#(ongoing)', reEncumber: true }
+      * def v = call createOrderLine { id: '#(poLineId)', orderId: '#(orderId)', fundId: '#(fundId)' }
+
+
+      # Open and unopen the order
+      * def v = call openOrder { orderId: '#(orderId)' }
+      * def v = call unopenOrder { orderId: '#(orderId)' }
+
+
+      ## Fiscal year rollover
+      Given path 'finance/ledger-rollovers'
+      And request
+        """
+        {
+          "id": "#(rolloverId)",
+          "ledgerId": "#(ledgerId)",
+          "fromFiscalYearId": "#(fyId1)",
+          "toFiscalYearId": "#(fyId2)",
+          "restrictEncumbrance": true,
+          "restrictExpenditures": true,
+          "needCloseBudgets": true,
+          "budgetsRollover": [
+            {
+              "rolloverAllocation": false,
+              "adjustAllocation": 0,
+              "rolloverBudgetValue": "None",
+              "setAllowances": false,
+              "allowableEncumbrance": 100,
+              "allowableExpenditure": 100
+            }
+          ],
+          "encumbrancesRollover": [
+            {
+              "orderType": "Ongoing",
+              "basedOn": "Remaining",
+              "increaseBy": 0
+            }
+          ]
+        }
+        """
+      When method POST
+      Then status 201
```

Comment:
seems like we need reusable feature for fiscal year rollover https://github.com/folio-org/folio-integration-tests/blob/98a7b08122b840292fa80164046b89abf4aa5b61/acquisitions/src/main/resources/thunderjet/mod-orders/features/remove_linked_invoice_lines_fund_distribution_encumbrance_reference.feature#L137